public-api/libproject/devstart.rb

```diff
@@ -341,6 +342,7 @@ def run_all_tests(cmd_name, args)
 
 def run_integration_tests(cmd_name, *args)
   ensure_docker cmd_name, args
+  common = Common.new
```

Comment:
nit: rm, this is defined later on as well

Reply:
Done thanks.